specs/eventing/overview.md

```diff
@@ -1,97 +1,88 @@
 # Resource Types
 
-The API defines and provides a complete implementation for
-[Trigger](spec.md#kind-trigger), [Broker](spec.md#kind-broker),
-[Subscription](spec.md#kind-subscription) and abstract resource definitions for
-[Channels](spec.md#kind-channel).
+The Knative Eventing API provides primitives for two common event-processing
+patterns (credit to James Urquhart for the formulation):
 
-With extensibility and composability as a goal of Knative Eventing, the eventing
-API defines several resources that can be reduced down to well understood
-contracts. These eventing resource interfaces may be fulfilled by other
-Kubernetes objects and then composed in the same way as the concrete objects.
-The interfaces are ([Addressable](interfaces.md#addressable),
-[Callable](interfaces.md#callable)). For more details, see
-[Interface Contracts](interfaces.md).
+* Point-to-point asynchronous communication ([`messaging.knative.dev`](#messaging))
 
-- A **Trigger** describes a filter on event attributes which should be delivered
-  to an _Addressable_.
+* Content-based event routing ([`eventing.knative.dev`](#eventing))
 
-- A **Broker** provides a bucket of events which can be selected by attribute.
+The other two patterns James identifies are log-stream processing and complex
+workflows; these are not currently addressed by Knative Eventing.
 
-<!-- https://drive.google.com/open?id=1CXRvT2g6sxk6-ZrwYcSf2BahCNVlLTLNZkm-laQitMg -->
+In addition to the primitives needed to express the above patterns, Knative
+Eventing defines two [_interface contracts_](#interface-contracts) to allow
```

Comment:
nit, here we say two interface contracts, but we only defined Addressable. If you think my suggestion about adding Destination is sane, then you do not have to change this :)